### DISCUSSION TITLE
May 18th 2022 - Identified Contacts

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [merchantprotocol/theadslab#56] [Announcements]
Hey All!

## Yes some things are "broken". 

I'm aware of a few areas that are not functioning as desired, regardless of any explanation, these items are not functioning as we would expect them to be.

- google sheet integration is not working
- "identified contacts" page is throwing an error
- and now I'm hearing about the webhook not working

## Why is all of this broken?

We just rolled out a change to how contact profiles are compiled and displayed. Previously we were delivering contact profiles is data silos, most easily understood from the google sheet integration where you had all of the data in separate tabs.

The new webhook now provides all of our data in a single record. And this is precisely the upgrade we've rolled out. All of the data is being combined into a single record so that you can more easily access it.

This has created a problem, in that it has broken the google sheet integration, it has caused an overly excessive compute requirement on the identified contacts page and now I have no idea why the webhook is not firing, if that is indeed the case for everybody.

### The webhook should be your primary method

I will ensure that the webhook is the top priority so that you can get all of the data out of the system and into a google sheet or other program of your choice. The webhook will be my top priority. 

If you don't like the zapier pricing to capture the webhook, I suggest you switch to integromat or even rudderstack which offer better pricing if you're unable to program something custom to capture the webhook.

### The identified contacts tab is second

My second priority will be the identified contacts tab to allow you to see all of the identified contact information. I will come out with a couple other reports in here that will allow you to see everything that is happening and to past events so you can access all of your data.

## Here's what you should remember

Even though we might go through these bumps and dips, we have captured your visitors identification and when things are working you will be able to access identified contacts for periods of time when things "were not working".

## My apologies

Sincerely, I apologize if this has caused a hicup in your business. I'm working round the clock to deliver the highest number of matches possible, and as consistently as we possibly can.

**Comment by abadger13** [comment]
Hey Jonathon -- thanks so much for taking the time on the update. I get it, and it's really clear now what you're doing. Much appreciated.
